src/components/RecipeModal/RecipeModal.jsx

```diff
@@ -8,6 +8,23 @@ export default function RecipeModal(props) {
         return null
     }
     const { recipeInfo, ...modalProps } = props
+    var validMins = true
```

Comment:
You can condense validMins, validServings, validIngred, and validSteps.
const validMins = !(recipeInfo.readInMinutes == undefined)

Reply:
Agreed. This would make this much easier to read.